src/AdminConsole/Pages/Organization/Settings.cshtml.cs

```diff
@@ -61,7 +67,7 @@ public async Task<IActionResult> OnPostDeleteAsync()
             }
         }
 
-        var isDeleted = await _dataService.DeleteOrganizationAsync(Organization.Id);
+        var isDeleted = await _dataService.DeleteOrganizationAsync(Id);

```

Comment:
I'm also wondering why the Delete call needs a ID at all, since it should probably used the one it has internally?

Reply:
I can refactor that bit separately